umap/static/umap/locale/cs_CZ.json

```diff
@@ -37,45 +37,45 @@
     "Icon shape": "Tvar ikony",
     "Icon symbol": "Symbol ikony",
     "Inherit": "výchozí",
-    "Label direction": "Směr popisku",
-    "Label key": "Tlačítko popisku",
-    "Labels are clickable": "Popisky jsou klikatelné",
+    "Label direction": "Label direction",
+    "Label key": "Label key",
+    "Labels are clickable": "Labels are clickable",
```

Comment:
Many localized strings in this file are replaced by English. Please do not apply it.

Reply:
Thanks Vladimir for checking my PR. I will have to check what is wrong with transifex.

Reply:
Can you please try to fix translations in transifex?
https://www.transifex.com/yohanboniface/umap/dashboard/

Reply:
I confirm translations are KO in transifex. can you please try to fix transifex?
![image](https://user-images.githubusercontent.com/6597721/74656682-ec4e9400-518e-11ea-9887-ef3fb45c91f5.png)

Reply:
@Binnette you should be able to push back old translations to Transifex with something like this: `tx push -t -l cs_cz`

Reply:
Done. I pushed the translation with command: `tx push -t -l cs_CZ`